src/output.js

```diff
@@ -90,8 +90,6 @@ class Debugger {
       this.capture();
       fun.call();
       return this.capturedMessages;
-    } catch (e) {
```

Comment:
I installed eslint@7, some packages related to eslint rule may be also updated, so I get 
`error  Unnecessary catch clause  no-useless-catch`